at_end2end_test/test/cached_key_test.dart

```diff
@@ -126,6 +126,62 @@ void main() {
     expect(response, contains('data:$value'));
   }, timeout: Timeout(Duration(seconds: 100)));
 
+  /// The purpose of this test verify the following:
+  /// 1. Share a key from atsign_1 to atsign_2 with ttr
+  /// 2. lookup from atsign_2 returns the correct value
+  /// 3. Update the existing key to a new value
+  /// 4. lookup with bypass_cache set to true should return the updated value
+  /// 5. lookup with bypass_cache set to false should return the old value
+  test('update-lookup verb passing bypass_cache ', () async {
+    ///Update verb on atsign_1
+    var value = 'user_1';
+    await sh1
+        .writeCommand('update:ttr:1000:$atSign_2:username$atSign_1  $value');
+    String response = await sh1.read();
+    print('update verb response : $response');
+    assert(
+        (!response.contains('Invalid syntax')) && (!response.contains('null')));
+
+    ///lookup verb alice  atsign_2
+    await sh2.writeCommand('lookup:username$atSign_1');
+    response = await sh2.read();
+    print('lookup verb response : $response');
+    expect(response, contains('data: $value'));
+
+    // TODO
```

Comment:
This can't simply be in the code flow, as it has to be in the GitHub Actions workflow. So the test needs to run up to this point, and then the action SSHs into the CICD VM and stops the secondary and then we start running another test that picks up from there etc.? @kalluriramkumar should be able to help, as he's familiar with the workflow definition.

Also how long do we expect these tests to take? If we're waiting around for minutes then that probably blows the 10m target for our commit cycle.

Reply:
@cpswan can you re-review please so Purnima can merge if you're happy?